### ISSUE TITLE
Accessing file details causes an error

### ISSUE THREAD

**Issue by lkiesow** [minio/console#754]
Go to any bucket, upload a file, then enter the file details and you will get to the details view but are immediately presented with an error:

![Screenshot from 2021-05-17 23-18-42](https://user-images.githubusercontent.com/1008395/118558139-8290b180-b766-11eb-9133-d261a04dcada.png)

**Comment by dvaldivia** [comment]
can you check the console logs to see if the error message is more clear regarding what went wrong, I'm suspecting one of the subcalls (to fetch retention, tags or something else) is failing

**Comment by dvaldivia** [comment]
I cannot replicate it with a plain bucket, does your `test2` bucket has any special features? (versioning, object locking, retention)

**Comment by lkiesow** [comment]
I don't think it has. Let me try again with a new bucket…

Meanwhile, here are the errors:

backend

```
2021-05-17 23:38:59.363894 I | 2021/05/17 23:38:59 error.go:52: user_objects.go:52: original error: A header you provided implies functionality that is not implemented
```

browser console:

```
XHRGEThttp://[::]:9090/api/v1/buckets/test2/objects?prefix=cut-marks.json&with_versions=true
[HTTP/1.1 500 Internal Server Error 3ms]

	
GET
	http://[::]:9090/api/v1/buckets/test2/objects?prefix=cut-marks.json&with_versions=true
Status500
Internal Server Error
VersionHTTP/1.1
Transferred300 B (61 B size)
Referrer Policystrict-origin-when-cross-origin

    	
    Content-Length
    	61
    Content-Type
    	application/json
    Date
    	Mon, 17 May 2021 21:38:59 GMT
    Server
    	MinIO Console
    X-Content-Type-Options
    	nosniff
    X-Frame-Options
    	DENY
    X-Xss-Protection
    	1; mode=block
    	
    Accept
    	*/*
    Accept-Encoding
    	gzip, deflate
    Accept-Language
    	en-US,en;q=0.5
    Cache-Control
    	no-cache
    Connection
    	keep-alive
    Cookie
    	token=AOpYRZAjrDP+zqYbzrC0L/RKXuoYVH5FflVFOCRm1seGg14MzIvw51gY7qKJTCJoo5JguNwZKFoYZjOflkO6GuV7kutZ5jPRxm0imxY0GkUbcgGmZ5JiqnM68M89ijBSB4OD5KXf+SIIQolo8n//fOkp2dRS7TPYtFi49P3J9mCSio5SJb/UnHZW8KZ4IGdfe3iU433BenK8UIMoTPaimbbr3gY1J077M+DeeULnv5ppHzFVnxG5SBf9ESon0hdsrOjdGngTGc7k2vVUQ5qbAKfyQ+D/gU/++Xg/Af7xKMEZtvX9EDWRKA/0yXTB/EFBtrj1PuxfQ9CNaRUw2DYArK3fukoHILCcIe+obFkN5HZJTd98DanCMnzWFkjwKGF7BfLYtb1++m0XB8HDC8cRhvgHhSsEbmiy9wZwdlmdB2KJcA7j+8CYXpj43rypBiDFBSvqDjr5IxDYMUozCaFe0I9vRieBBUyEwQmOFKz9RtXIXpQ2IjaA9/RO3VXGi1TqnGzPDH5TY9Bsv5tcb39807dGyPZeE7hTvD+etzI7CIrV1jGPef8X9yCfEvP3k7NVbjLrmeOtkSO8lxsGruue4PEr0s5bT0EQXz/km+zHA8aXzm8CnOcRmMA55SjOzeIpuh+MpES9O9Ehe/myNlLGfg4mJxn2LspJw+5UsDqFavu6Iko4Cg==
    DNT
    	1
    Host
    	[::]:9090
    Pragma
    	no-cache
    Referer
    	http://[::]:9090/object-browser/test2/cut-marks.json
    User-Agent
    	Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0

**Comment by lkiesow** [comment]
Still the same issue with a new bucket with no special features. Maybe one of my previous test (creating roles with special characters, …) has permanently broken something. I'll try resetting the whole system later and see if the issue persists.

**Comment by dvaldivia** [comment]
perhaps this is happening because you are running a standalone MinIO, is that the case @lkiesow ?

**Comment by dvaldivia** [comment]
I think that error will happen when we check for retention for a given object due to the error message 
```
A header you provided implies functionality that is not implemented
```

**Comment by lkiesow** [comment]
> perhaps this is happening because you are running a standalone MinIO, is that the case @lkiesow ?

Yes, it's a standalone installation with more or less everything left to the default settings I just spun up for testing _console_.

**Comment by danielnieto** [comment]
I have the same issue, I just wanted to "try" MinIO with Docker, I ran:

```
sudo docker run \
  -p 9000:9000 \
  -p 9001:9001 \
  --name minio1 \
  -e "MINIO_ROOT_USER=user" \
  -e "MINIO_ROOT_PASSWORD=password" \
  -v .:/data \
  minio/minio server /data --console-address ":9001"
  ```
  
  And then login into the console, create a simple bucket (with all features OFF) and upload a file, when trying to see the file, it fails:
  
`original error -> (user_objects.go:51: A header you provided implies functionality that is not implemented)`  

I think the problem is that the frontend is trying to get the object with "versions": `api/v1/buckets/test/objects?prefix=_.jpeg&with_versions=true` even though I've set the bucket to NOT use versions... If I remove the querystring param it works....

**Comment by bexsoft** [comment]
Fixed in https://github.com/minio/console/pull/880

**Comment by danielnieto** [comment]
@bexsoft thanks for fixing it! Que chingon!
